### PR TITLE
Do not copy UWP behavior with ListBoxItem horizontal alignment

### PR DESCRIPTION
**Pull request by maxkatz6**
## What is the current behavior?
In Default theme default ListBoxItem horizontal alignment is Stretch. Same as in WPF.
In Fluent theme we copy UWP behavior and set Left value instead. [Note, newer WinUI versions use Stretch as well](https://github.com/microsoft/microsoft-ui-xaml/pull/3914).

## Why is it a problem?
https://stackoverflow.com/questions/43094395/how-to-stretch-listviewitem-in-listview
https://github.com/microsoft/microsoft-ui-xaml/issues/1402

## What is the updated/expected behavior with this PR?
Do not override default value in theme. Which makes it "Stretch" value (HorizontalAlignment enum default value).

## Breaking changes

Technically it is a styling breaking change.

### CODE REVIEW

**Comment by robloo** [comment]
Yea, anyone using WinUI 2 library or WinUI 3 will have stretch as the default. That was a surprisingly difficult conversation and it represents perfectly everything wrong with WinUI development these days: https://github.com/microsoft/microsoft-ui-xaml/issues/1402

Anyway, seeing this simple commit with no fuss because everyone knows its better is so refreshing.

**Review by jmacato — APPROVED** [review]
Yeah this makes more sense :) Let's wait for @danwalmsley 's approval before merging

**Comment by robloo** [comment]
Here is my analysis of breaking changes. Bottom line, extremely minimal.

https://github.com/microsoft/microsoft-ui-xaml/issues/1402#issuecomment-737396498

**Comment by jmacato** [comment]
got approval from @danwalmsley :)

**Pull request merged**
